client/src/components/Home.tsx

```diff
@@ -241,73 +243,85 @@ function Home() {
       {user && <Releases />}
       {user && posts.length > 0 && (
         <>
-          <SectionHeader>
+          <SectionHeader
+            className={css`
+              background-color: var(--mi-light-background-color) !important;
+            `}
+          >
             <WidthContainer variant="big" justify="center">
               <h5>{t("latestCommunityPost")}</h5>
             </WidthContainer>
           </SectionHeader>
-          <WidthContainer variant="big" justify="center">
-            <div
-              className={css`
-                margin-top: 1rem;
-                display: flex;
-                flex-wrap: wrap;
-                justify-content: space-between;
 
-                a {
-                  width: 32%;
-                }
-
-                @media screen and (max-width: ${bp.xlarge}px) {
-                  padding: var(--mi-side-paddings-xsmall);
-                }
+          <div
+            className={css`
+              background-color: var(--mi-light-background-color);
 
-                @media screen and (max-width: ${bp.medium}px) {
-                  flex-direction: column;
+              a {
```

Comment:
are we sure we want to apply this to all links inside this div container?

Reply:
Yeah cause it's all stuff that you click on anyway. Like links were broken for stuff where there's a link inside a sentence, but for article titles, album titles etc. There's just no need to accentuate them more (they all have underline on hover anyway). It's just important to keep links colored in markdown, blogposts etc. Wherever it's not clear that some things in the text/UI are clickable.

Reply:
Then shouldn't this be universal? In styles.ts?

Reply:
It's just for home page, it's because basically on home page it's ALL links. Everything is clickable so it's the opposite for accessibility it's worse if it's all colored. But then as soon as it's not all link it goes back to default rule - (links have a different color + underlined on hover)

Reply:
(see footer where links are displayed normally)